install-dev/controllers/http/configure.php

```diff
@@ -150,14 +160,17 @@ public function processUploadLogo()
                     list($src_width, $src_height, $type) = getimagesize($tmp_name);
                     $src_image = ImageManager::create($type, $tmp_name);
                     $dest_image = imagecreatetruecolor($src_width, $src_height);
+                    // @phpstan-ignore-next-line
                     $white = imagecolorallocate($dest_image, 255, 255, 255);
+                    // @phpstan-ignore-next-line
                     imagefilledrectangle($dest_image, 0, 0, $src_width, $src_height, $white);
+                    // @phpstan-ignore-next-line
                     imagecopyresampled($dest_image, $src_image, 0, 0, 0, 0, $src_width, $src_height, $src_width, $src_height);
+                    // @phpstan-ignore-next-line
                     if (!imagejpeg($dest_image, _PS_ROOT_DIR_ . '/img/logo.jpg', 95)) {
-                        $error = $this->trans('An error occurred during logo copy.', [], 'Install');
+                        $error = $this->translator->trans('An error occurred during logo copy.', [], 'Install');
```

Comment:
What even is this code ? 😆  I dont find where we can upload a logo in the install process.

It's probably dead code ping @kpodemski @Quetzacoalt91 any memory of the install process supposed to allow logo upload?

Reply:
Yeah I remember such a feature in very old versions of PrestaShop! It was maybe PS 1.4

Reply:
@sarahdib has a lot of archives 

![image](https://user-images.githubusercontent.com/1533248/144585878-d1428186-9acf-4f07-b09b-3fe3b6c0bac9.png)

Reply:
THE ARCHIVIST! :file_cabinet: